headers.yaml

```diff
@@ -19,7 +19,7 @@ X-Rate-Limit-Remaining:
   schema:
     type: integer
 X-Rate-Limit-Reset:
-  description: The number of seconds left in the current period
+  description: Timestamp of end of current time period as UNIX timestamp
```

Comment:
link the relevant intro section

Reply:
What you mean by link? Should I use a ref here?

Reply:
```suggestion
  description: Timestamp of end of current time period as UNIX timestamp, see [Rate Limiting](#overview--rate-limiting)
```